lunch-application/src/theme/Home.vue

```diff
@@ -32,15 +32,15 @@
           </tr>
         </thead>
         <tbody>
-          <tr v-for="row in posts">
+          <tr v-for="row in restaurantData">
             <td scope="row"></td>
             <td scope="row">{{row.restaurantName}}</td>
           </tr>
         </tbody>
       </table>
     </div>
-    <button type="button" class="btn">Edit Top 5</button>
-    <input class="btn btn-primary" type="submit" value="Submit">
+    <button type="button" class="btn" >Edit Top 5</button>
+    <input class="btn btn-primary" type="submit" value="Submit" v-on:click="lunchAlert()" >
```

Comment:
this is a good time to rename the method now that you know what you want it to do